Makefile

```diff
@@ -308,12 +439,9 @@ $(addprefix .build-artefacts/annotated/, $(SRC_JS_FILES) src/TemplateCacheModule
 	${PYTHON_CMD} .build-artefacts/python-venv/bin/pip install "gspread==0.2.2"
 	touch $@
 
-.build-artefacts/ol-requirements-installation.timestamp: .build-artefacts/python-venv
```

Comment:
Than maybe it's not needed anymore. ol3 build system is quite in flux :) Thanks alot for the tests.